theorems/T000742.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: T000742
+if:
+  and:
+    - P000112: true
+    - P000164: true
+    - P000006: true
+then:
+  P000215: true
+refs:
+- doi: 10.1007/978-1-4615-7819-2
+  name: Rings of Continuous Functions (Gillman & Jerison)
+---
+
+Since $X$ is {P112} exists a continuous bijection $f:X\to Y$ where $Y$ is a metrizable space. From {T382} it follows that because {P53} is a hereditary property, $Y$ is {P215}. From corollary 8.18 of {{doi:10.1007/978-1-4615-7819-2}} $X$ is {P215}.
```

Comment:
```suggestion
Since $X$ is {P112}, there exists a continuous bijection $f:X\to Y$ where $Y$ is a {P53} space. Necessarily $Y$ has {P164}.
Every {P53} space is {P7} and {P194}
[(Explore)](https://topology.pi-base.org/spaces?q=Metrizable%2B%7ESubmetacompact).
So every subspace of $Y$ satisfies the hypotheses of {T382},
and hence is {P162}.
By Corollary 8.18 of {{doi:10.1007/978-1-4615-7819-2}}
every subspace of $X$ is {P162}.
```
Same thing as what you had, with just a few things more explicit.
(You can cut and past in the pi-base "preview" window to compare)